### DISCUSSION TITLE
failed to fetch when i save a change

### DISCUSSION THREAD

**Discussion by pinale** [supabase/supabase#30700] [Questions] [answered]
On a fresh new project, create a teble, save it, ok done.

when i try to modify a table with chrome (adding a field) and press "save"
i got "failed to fetch" and in console (F12)

GET https://api.supabase.com/v1/projects/kmisqvziodlbaehoyedd/custom-hostname
404 bad request
{
    "message": "Project xxxxxxxxxxxxx is not allowed to set up custom domain; please contact Supabase support."
}

obviously i've no custom domain on the free tier 

the problem occours only in chrome (tried with edge and works).
On Chrome i've clean all internet temporal files, clean all local storage type(local/session/Indeddb/cookies... storage) in the "F12->Application tab"

**Comment by GaryAustin1** [comment]
I would hard refresh your browser.  Try a different browser.  Restart the instance (if you can).
Then contact support like it says.

**Reply by pinale** [reply]
the problem occours only with chrome and i've clean all possible.
What about "Restart  instance"? i'm using the supabase free cloud service (it's not running local) is it possible to restart? how?

**Reply by GaryAustin1** [reply, marked as answer]
Please check for an extension in Chrome involving CORS.  There is one that blocks PATCH commands.
https://github.com/orgs/supabase/discussions/17296

You can restart the instance in the infrastructure settings but if it works from another browser that would not have impact.

**Reply by pinale** [reply]
as you suggested, the guilty was the extension.
Thanks 💯 
[CORS Unblock](https://chromewebstore.google.com/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino)
